packages/core/src/Cardano/types/Cip1854ExtendedAccountPublicKey.ts

```diff
@@ -0,0 +1,36 @@
+import * as BaseEncoding from '@scure/base';
+import { Bip32PublicKeyHex } from '@cardano-sdk/crypto';
+import { InvalidStringError, OpaqueString, assertIsBech32WithPrefix, assertIsHexString } from '@cardano-sdk/util';
+
+const MAX_BECH32_LENGTH_LIMIT = 1023;
+const bip32PublicKeyPrefix = 'acct_shared_xvk';
+
+export type Cip1854ExtendedAccountPublicKey = OpaqueString<'Cip1854PublicKey'>;
```

Comment:
Since it's not in the name, please add a doc comment about what encoding this type uses
```suggestion
/**
 * bech32-encoded... 
 */
export type Cip1854ExtendedAccountPublicKey = OpaqueString<'Cip1854PublicKey'>;
```

Reply:
Fixed on 7b583d8